docs/guide/node_in_cloud.md

```diff
@@ -0,0 +1,152 @@
+2. Make sure the @tensorflow/tfjs-node dependency is @1.2.4 or newer version.
+
+### Build your app and run it locally
+
+1. Run the `npm install` command in your local directory to install the dependencies that are declared in the `package.json` file. You should be able to see the tfjs-node package is installed and libtensorflow is downloaded.
+
+```
+$ npm install
+> @tensorflow/tfjs-node@1.2.5 install mnist-node/node_modules/@tensorflow/tfjs-node
+> node scripts/install.js
+
+CPU-linux-1.2.5.tar.gz
+* Downloading libtensorflow
+[==============================] 22675984/bps 100% 0.0s
+* Building TensorFlow Node.js bindings
+```
+
+2. Train the model locally by running `npm start`.
+
+```
+$ npm start
+> tfjs-examples-mnist-node@0.1.0 start /mnist-node
+> node main.js
+
+2019-07-30 17:33:34.109195: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
+2019-07-30 17:33:34.147880: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3492175000 Hz
+2019-07-30 17:33:34.149030: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x52f7090 executing computations on platform Host. Devices:
+2019-07-30 17:33:34.149057: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
+
+Total params: 594922
+Trainable params: 594922
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1 / 20
+Epoch 1 / 20
+========================>----------------------------------------------------------------------------------: 35.5
+```
+
+3. Make sure you ignore build artifacts, such as node_modules, in your .gitignore file.
+
+### Create and deploy the heroku app
+
+1. Create a new app on heroku website
+2. Commit your change and push to heroku master
+
+```
+$ git init
+$ heroku git:remote -a your-app-name
+$ git add .
+$ git commit -m "First Commit"
+$ git push heroku master
+```
+
+3. In the build logs, you should be able to see the tfjs-node package downloading the TensorFlow C Library and loading TensorFlow Node.js native addon:
+
+```
+remote: -----> Installing dependencies
+remote:        Installing node modules (package.json)
+remote:
+remote:        > @tensorflow/tfjs-node@1.2.5 install /tmp/build_de800e169948787d84bcc2b9ccab23f0/node_modules/@tensorflow/tfjs-node
+remote:        > node scripts/install.js
+remote:
+remote:        CPU-linux-1.2.5.tar.gz
+remote:        * Downloading libtensorflow
+remote:
+remote:        * Building TensorFlow Node.js bindings
+remote:        added 92 packages from 91 contributors and audited 171 packages in 9.983s
+remote:        found 0 vulnerabilities
+remote:
+```
+
+In the process logs on heroku you should be able to see the model training logs:
```

Comment:
"In the process logs on Heroku, you should be able to see the model training logs."

Reply:
Done.